### DISCUSSION TITLE
SMPlayer YouTube video and audio are not played in the correct format.

### DISCUSSION THREAD

**Discussion by Organarh** [smplayer-dev/smplayer#1003] [General]
I launch the YouTube link (network - use adaptive streams up to 4k, if possible 60k....)
![1](https://github.com/smplayer-dev/smplayer/assets/100313332/a1189e89-cd51-4670-ba2c-2f4722c0e1ff)`
Video and audio are not played in the correct format.

1. Initial Video Stream
Resolution: 1920 x 1080
Aspect ratio: 1.77778
Format: vp9
Bitrate: 1350 kbps
Frames per second: 60.000004
Selected codec: vp9

Initial Audio Stream
Format: opus
Bitrate: 142 kbps
Rate: 48000 Hz
Channels: 2
Selected codec: opus

Very high laptop load. Intel® Core™ i5-2410M Intel® HD Graphics 3000
I turn it off (network - use adaptive streams up to 4k, if possible 60k....)
Initial Video Stream
Resolution: 640 x 360
Aspect ratio: 1.77778
Format: h264
Bitrate: 0 kbps
Frames per second: 30.000000
Selected codec: h264

Initial Audio Stream
Format: aac
Bitrate: 95 kbps
Rate: 44100 Hz
Channels: 2
Selected codec: aac
This happens in Windows and Linux. 
smplayer (.ini) 
[streaming]
streaming_type=1
ytdl_quality=bestvideo[height<=?1080]+bestaudio/best[height<=?1080]
streaming\youtube\resolution=5
streaming\youtube\preferred_user_agent=
streaming\youtube\ytdl_bin=
streaming\youtube\override_format=
streaming\youtube\use_dash=false
streaming\youtube\use_60fps=true
streaming\youtube\use_av1=false
Tried it  "codec:h264,height:1080"[codec:h264]+bestaudio[ext=m4a]/best[ext=mp4]/best"' to no avail.
 Where in the player settings can you configure it to work correctly?

**Comment by smplayer-dev** [comment]
I can't reproduce this problem with the videos I tested. Can you provide a youtube url where this happens?

**Reply by Organarh** [reply]
>I can't reproduce this problem with the videos I tested. Can you provide a youtube url where this happens?

This is a problem with my laptop Samsung np-rc720-s02ru (Intel® Core™ i5-2410M Intel® HD Graphics 3000 NVIDIA GeForce GT 520M) and not with the video, it reproduces all videos from YouTube like this. Others most likely do not have such problems. That’s why I and I ask for help why the player does not correctly detect the hardware and launch the video in vp9 format, sound opus.
